README.md

```diff
@@ -2,33 +2,75 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-accounts-cyhy/workflows/build/badge.svg)](https://github.com/cisagov/cool-accounts-cyhy/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
-
-## Usage ##
-
-```hcl
-module "example" {
-  source = "github.com/cisagov/cool-accounts-cyhy"
-
-  aws_region            = "us-west-1"
-  aws_availability_zone = "b"
-  subnet_id             = "subnet-0123456789abcdef0"
-}
-```
-
-## Examples ##
-
-- [Basic usage](https://github.com/cisagov/cool-accounts-cyhy/tree/develop/examples/basic_usage)
+This project contains Terraform code to perform the initial configuration of a
+COOL Cyber Hygiene (CyHy) account. This Terraform code creates and configures
+the most basic resources needed to build out services and environments.
+
+It creates an IAM role that allows sufficient permissions to provision all AWS
+resources in this account. This role has a trust relationship with the COOL
+users account.
+
+## Bootstrapping this account ##
```

Comment:
I'm not really sure where we are on this but it might be worth mentioning any [cisagov/cool-accounts](https://github.com/cisagov/cool-accounts) dependencies (mainly the `terraform` account from what I can see).

Reply:
Good catch- please have a look at what I added in https://github.com/cisagov/cool-accounts-cyhy/pull/1/commits/7e1ef6407de049b72eef567dcac39b791e5a4d25.